renku/core/dataset/providers/renku.py

```diff
@@ -514,7 +514,7 @@ def _fetch_dataset(self, client_dispatcher: IClientDispatcher, database_dispatch
             database_dispatcher.pop_database()
             client_dispatcher.pop_client()
 
-        provider_dataset.data_dir = get_dataset_data_dir(self._remote_client, provider_dataset.name)
+        provider_dataset.data_dir = provider_dataset.get_datadir(self._remote_client)
```

Comment:
We don't need this anymore; it was set so that we have the dataset's datadir in the model. Now that we have it we can directly call `dataset.get_datadir()` instead of this.